src/redux/actions/authActions.js

```diff
@@ -1 +1,42 @@
-import * as types from '../types';
+import { LOGIN_SUCCESS, LOGIN_FAIL, LOGOUT, SET_LOADING } from '../types';
+import axios from 'axios';
+import store from '../store/index';
+import ActionButton from 'antd/lib/modal/ActionButton';
+
+console.log(store.getState());
+const reduxState = store.getState();
+//Login User
+export const loginUser = accessToken => async dispatch => {
+  try {
+    setLoading();
```

Comment:
calling the `setLoading` action this way won't really `dispatch` the action. Actually this action is never getting called. Dispatch the `action type` from here itself directly.
This is also the reason for incorrect _loading_ settings on the button.

Reply:
Can we dispatch two action types together?

Reply:
Yes, you can use `dispatch` with `SET_LOADING` type instead to calling `setLoading()`